lib/bundler/cli/install.rb

```diff
@@ -185,5 +181,13 @@ def print_post_install_message(name, msg)
       Bundler.ui.confirm "Post-install message from #{name}:"
       Bundler.ui.info msg
     end
+
+    def check_for_options_conflicts
```

Comment:
boo. if we do it elsewhere, fine, but I'd prefer to see us raising some sort of `invalid command usage` error instead of all the `SystemExit`s